src/content/self-hosted/helm-configuration.mdx

```diff
@@ -485,6 +485,35 @@ regcredsManager:
       memory: 800Mi
 ```
 
+### resourceManager
+
+Use this section to configure the [Resource Manager](admin/resource-manager.mdx)
+
+- `enabled`: When set to `true`, the Resource Manager is installed. When Resource Manager is installed, Okteto creates a CronJob responsible for computing the cpu and memory recommendations for your cluster.
+- `schedule`: Defines the frequency at which resource recommendations are calculated, using the cron syntax.
+- `deletePeriod`: Specifies the retention period for the recommendations. If a service is no longer deployed in your cluster, its recommendation will be deleted after the specified deletePeriod.
+- `recommendations.weight`: A floating-point number between 0 and 1 used to stabilize CPU and memory recommendation calculations. A higher value (closer to 1) gives more weight to historical resource usage trends, making the Resource Manager less responsive to recent changes or spikes. A lower value (closer to 0) makes it more reactive to recent usage patterns, including spikes. The default value is `0.9`, which makes the Resource Manager conservative, favoring existing trends and attempting to ignore minor fluctuations.
```

Comment:
@codyjlandstrom can you proof read the initial draft, and tell me your thoughts?

With @pchico83 we are not 100% sure on the naming for `weight` and `correction`. 

1. Is it clear what they are for?
2. do you have any better names?

🙏

Reply:
The initial draft sounds great, @andreafalzetti.

1. I actually like these names and think they are pretty clear
2. `correction` could potentially be "compensation" but I think both are easily understandable. Users configuring these will have to refer to the documentation to understand the values regardless, so I don't think there is much difference in the name